.github/workflows/main.yaml

```diff
@@ -98,11 +98,9 @@ jobs:
           echo "check_suite_id=${{ github.event.check_run.check_suite.id }}" >> $GITHUB_ENV
         else
           echo "Retrieving failed check runs for check suite ID: ${{ github.event.check_suite.id }}"
-          gh api \
-            -H "Accept: application/vnd.github+json" \
-            -H "X-GitHub-Api-Version: 2022-11-28" \
```

Comment:
I don't think that the API version is required, but the accept field maybe required to ensure the format of the output.

Reply:
I've readded them, bad copy paste